types/src/v28/blockchain.rs

```diff
@@ -13,7 +13,6 @@ use super::{GetBlockchainInfoError, Softfork};
 use crate::model;
 
 #[rustfmt::skip]                // Keep public re-exports separate.
-
```

Comment:
In a81e7c95517a5237990b7dea58ac6431e7efff4a:

Can you remove this unrelated change?

Reply:
We can just remove the whole attribute. Its meant to be on `pub use Foo` statements. It was incorrectly added in fcd96963